lib/src/zap/rdma/zap_rdma.c

```diff
@@ -105,7 +105,7 @@
 #ifdef CTXT_DEBUG
```

Comment:
@narategithub we should probably either make this ifdef DEBUG, or add CTXT_DEBUG to the github DEBUG build test. I really don't want to discover these things one at a time.

Reply:
sure. I should also make github `-DDEBUG` build job to build zap_rdma and zap_fabric (the library dependencies are available on CentOS7 repo). By default they aren't built.

Reply:
@tom95858 I put in various DEBUG flags in debug build github workflow in #509.